### PR TITLE
Sample details refactor/ details tab

### PR DESCRIPTION
**Pull request by deepsidhu85**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Connected SampleDetailViewer -> details tab to server. Added functionality to edit sample details fields.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
~* [ ] Tests added (or description of how to test) for any new features.~
~* [ ] User documentation updated for UI or technical changes.~

### CODE REVIEW

File: src/main/webapp/yarn.lock [hunk 1]
```diff
@@ -7145,7 +7164,7 @@ fsevents@^2.1.2:
     bootstrap-daterangepicker: ^3.0.3
     browserslist: ^4.16.6
     chroma-js: ^1.3.4
-    chromedriver: 92.0.0
+    chromedriver: ^95.0.0
```
Comment (joshsadam) [inline src/main/webapp/yarn.lock:7167]: We need to get this into development and pushed up through the branches.

File: src/main/webapp/resources/js/components/samples/sampleSlice.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { createSlice } from "@reduxjs/toolkit";
+
```
Comment (joshsadam) [inline src/main/webapp/resources/js/components/samples/sampleSlice.js:2]: Are you using this anywhere?

**Review by joshsadam — CHANGES_REQUESTED** [review]

File: src/main/webapp/resources/js/components/samples/SampleDetailViewer.jsx [hunk 3]
```diff
@@ -87,3 +88,23 @@ export function SampleDetailViewer({ sampleId, removeSample, children }) {
...
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function SampleDetailViewer({ sampleId, removeSample, children }) {
+  return (
+    <Provider store={store}>
+      <DisplaySampleDetails
+        sampleId={sampleId}
+        removeSample={removeSample}
+        children={children}
```
[the remark this answers is not in the document]
Comment (deepsidhu85) [reply src/main/webapp/resources/js/components/samples/SampleDetailViewer.jsx]: Moved into parent component in d344f6a69187dd35b5345da11b2228c34262a2a6

File: src/main/webapp/resources/js/components/samples/components/SampleInfo.jsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
-import { List, Space, Typography } from "antd";
+import { DatePicker, List, notification, Space, Typography } from "antd";
 import { CalendarDate } from "../../CalendarDate";
```
[the remark this answers is not in the document]
Comment (deepsidhu85) [reply src/main/webapp/resources/js/components/samples/components/SampleInfo.jsx]: Removed in d344f6a69187dd35b5345da11b2228c34262a2a6

File: src/main/webapp/yarn.lock (hunk 1 above)
Comment (deepsidhu85) [reply src/main/webapp/yarn.lock:7167]: I agree!

File: src/main/webapp/resources/js/components/samples/sampleSlice.js (hunk 2 above)
Comment (deepsidhu85) [reply src/main/webapp/resources/js/components/samples/sampleSlice.js:2]: Not yet but will be used in the SampleMetadata component. I just added it as I was setting up the redux stuff for this refactor. If I end up not using it I will remove it in the next PR

**Comment by deepsidhu85** [comment]
This branch got corrupted on local machine and has been replaced with PR #1130

**Pull request closed without merging**